src/extension.ts

```diff
@@ -1224,26 +1229,27 @@ export class ExtensionManager implements vscode.Disposable {
     }
 
     clean(folder?: vscode.WorkspaceFolder) {
-        telemetry.logEvent("clean");
+        telemetry.logEvent("clean", { all: "false"});
         return this.build(folder, 'clean', undefined, false);
     }
 
     cleanAll() {
-        telemetry.logEvent("clean");
+        telemetry.logEvent("clean", { all: "true"});
         return this.buildAll(['clean']);
     }
 
     cleanRebuild(folder?: vscode.WorkspaceFolder) {
-        telemetry.logEvent("clean");
+        telemetry.logEvent("clean", { command: "cleanRebuild", all: "false"});
```

Comment:
Let's just rename this event to "cleanRebuild" instead of keeping it as "clean".

Same with the clean rebuild all.